lib/reactor.js

```diff
@@ -33,6 +33,7 @@ import * as ruleComponents from './ruleComponents';
 import * as rules from './rules';
 
 class NullLogger {
+  always() {}
```

Comment:
In node-specific libraries, I _have_ often seen libraries allow you to pass in a log level. Since this library is intended for both node and browser usage, maybe you could make the option value be a log level instead of just `true` or `false`. For the log levels I think you should stick with those provided by the native `console` API.

Reply:
That makes perfect sense. I hadn't known there _were_ such a things as `console.`{`info`,`warn`,`error`}. 

I will do as you suggest.